pkg/ddc/alluxio/status.go

```diff
@@ -130,6 +130,12 @@ func (e *AlluxioEngine) CheckAndUpdateRuntimeStatus() (ready bool, err error) {
 			ready = true
 		}
 
+		// Update the setup time of Alluxio runtime
+		if ready && runtimeToUpdate.Status.SetupPeriodSeconds == nil {
```

Comment:
@yangyuliufeng I remember we have the similar function. Could you help @iluoeli refactor this implementation?

Reply:
> @yangyuliufeng I remember we have the similar function. Could you help @iluoeli refactor this implementation?

OK